atest/testdata/standard_libraries/string/format_string_template.txt

```diff
@@ -0,0 +1 @@
+Format String also {condition} files templates!
```

Comment:
Some tests for the format specification mini-language would be nice. At least some tests for the format spec, but possibly also for alignment and attribute/item access. Obviously no need to test the whole mini-language, but making sure the common and for us relevant cases actually work would be nice.

The attribute and item access test could use something like this:

```robotframework
*** Variables ***
&{USER}                 name=John Doe    email=john@example.com

*** Test Cases ***
Attribute access
    ${result} =    Format String    {user.name} <{user.email}>    user=${USER}
    Should Be Equal    ${result}    John Doe <john@example.com>

Item access
    ${result} =    Format String    {user[name]} <{user[email]}>    user=${USER}
    Should Be Equal    ${result}    John Doe <john@example.com>
```

Notice that I don't know do the above work but I expect they do!

Reply:
Added those tests. And they passed :)